docs/rpc/openapi.yaml

```diff
@@ -39,6 +39,17 @@ paths:
                 $ref: ./api/transaction/post-core-node-transactions-error.schema.json
               example:
                 $ref: ./api/transaction/post-core-node-transactions-error.example.json
+      parameters:
```

Comment:
Not for this line but...

Can you add what you want the CHANGELOG entry for this change to be? Either in the description or in the CHANGELOG file?